api/src/services/listing.service.ts

```diff
@@ -954,6 +970,20 @@ export class ListingService implements OnModuleInit {
       },
     );
 
+    //manually check for juris/listing mismatch since logic above is forcing admin permissioning
+    if (
+      (requestingUser?.userRoles?.isJurisdictionalAdmin &&
+        !requestingUser?.jurisdictions?.some(
+          (juris) => juris.id === storedListing.jurisdictionId,
+        )) ||
+      (requestingUser?.userRoles?.isPartner &&
+        !requestingUser?.listings?.some(
+          (listing) => listing.id === storedListing.id,
+        ))
+    ) {
+      throw new ForbiddenException();
+    }
```

Comment:
@mcgarrye This PR is waiting on https://github.com/bloom-housing/bloom/pull/4403 but wanted to flag this section above to you to get your thoughts since its connected to the tests we discussed. From what I can tell with the isAdmin overriding, we can't just rely on the permissionService.can to cover when a jurisdictional admin has the permissions to copy but attempts to do it in the wrong jurisdiction or partners on a forbidden listing since we'll pass isAdmin is true and avoid the other parts of the permissioning logic.